lib/pbench/test/functional/server/conftest.py

```diff
@@ -48,9 +56,17 @@ def register_test_user(server_client: PbenchServerClient):
 
 
 @pytest.fixture
-def login_user(server_client: PbenchServerClient, register_test_user):
+def login_user(
+    server_client: PbenchServerClient, oidc_admin: OIDCAdmin, register_test_user
+):
     """Log in the test user and return the authentication token"""
-    server_client.login("tester", "123456")
-    assert server_client.auth_token
-    yield
-    server_client.logout()
+    oidc_endpoints = server_client.endpoints["openid-connect"]
+    response = oidc_admin.user_login(
+        client_id=oidc_endpoints["client"], username="tester", password="123456"
+    )
+    auth_token = response.json()["access_token"]
+    assert auth_token
+    json = {"username": "tester", "auth_token": auth_token}
+    server_client.username = "tester"
+    server_client.auth_token = auth_token
+    yield JSONMap(json)
```

Comment:
The contents of `json` look like a simple Python dictionary to me...why are we returning it as a `JSONMap`?  (What's wrong with using a conventional Python dictionary?)

Reply:
`JSONMap` allows first level keys to be accessed as attributes, so `json.username` instead of `json["username"]`: it was an attempt at building a somewhat generalized "external resource" object representation. (One I'm not entirely sure was a success, but I may extend it later.)

But it's part of the _client library_, and we shouldn't be using it in the server code.

Reply:
I agree that `JSONMap` is a very cool widget!  I just don't think we should be using it if we don't need its special powers.

And, your point about which library it comes from is well-taken, although, this isn't, strictly speaking, _server code_ -- it's **test** code....

Reply:
So, @npalaska, I think @dbutenhof and I concur that you should be using a plain `dict` here instead of `JSONMap` (although we have different reasons)...why do you think you _should_ use the `JSONMap`?